README.md

```diff
@@ -4,7 +4,7 @@ _jsvu_ is the <b>J</b>ava<b>S</b>cript (engine) <b>V</b>ersion <b>U</b>pdater.
 
 _jsvu_ makes it easy to install recent versions of various JavaScript engines without having to compile them from source.
 
-[![](https://asciinema.org/a/rfS1M5ynKm1hGaBqJYJj0mGCi.png)](https://asciinema.org/a/rfS1M5ynKm1hGaBqJYJj0mGCi)
+[![jsvu Screenshot](https://rawgit.com/GoogleChromeLabs/jsvu/3bda9dd/screenshot.svg)](https://asciinema.org/a/rfS1M5ynKm1hGaBqJYJj0mGCi)
```

Comment:
Just to explain why I amended this: “jsvu Screenshot” is not useful alt text. In this case it’s [fine](https://html.spec.whatwg.org/multipage/images.html#alt) (and IMHO, _better_) to provide no alt text at all.